<launch>
    <arg name="img_topic" default="/head_xtion/rgb/image_raw"/>
    <arg name="sample_size" default="40"/>
    <arg name="wait_time" default="30"/>
    <arg name="publish_image" default="false"/>
    <arg name="save_mode" default="true"/>
    <arg name="interrupt_tasks" default= ""/>

    <arg name="machine" default="localhost" />
    <arg name="user" default="" />

    <machine name="$(arg machine)" address="$(arg machine)" env-loader="$(optenv ROS_ENV_LOADER )" user="$(arg user)" default="true"/>

    <node pkg="simple_change_detector" type="detector.py" name="change_detection" output="screen">
        <param name="img_topic" value="$(arg img_topic)" type="string"/>
        <param name="sample_size" value="$(arg sample_size)" type="int"/>
        <param name="wait_time" value="$(arg wait_time)" type="int"/>
        <param name="publish_image" value="$(arg publish_image)" type="bool"/>
        <param name="save_mode" value="$(arg save_mode)" type="bool"/>
        <param name="interrupt_tasks" value="$(arg interrupt_tasks)" type="string"/>
    </node>
</launch>
